package.json

```diff
@@ -28,6 +28,7 @@
     "@material-ui/icons": "^4.11.2",
     "@material-ui/lab": "^4.0.0-alpha.58",
     "@material-ui/styles": "^4.11.4",
+    "@octokit/core": "^4.0.4",
```

Comment:
Why not have a different `package.json` for the CI in another folder? It doesn't make sense to add dependencies to the dashboard that it doesn't need